sam/models/mlp_model.py

```diff
@@ -537,7 +538,7 @@ def quantile_feature_importances(
                 "This method is currently not implemented " "for multiple targets"
             )
 
-        from eli5.permutation_importance import get_score_importances
+        # from eli5.permutation_importance import get_score_importances
```

Comment:
Delete

Reply:
Remove also references to eli5 in the docstrings

Reply:
And also a line below with the comment "# use eli5 to compute feature importances:" which for some reason I cannot add a comment to